captcha/widgets.py

```diff
@@ -72,8 +72,8 @@ class ReCaptchaV3(ReCaptchaBase):
 
     def __init__(self, api_params=None, *args, **kwargs):
         super().__init__(api_params=api_params, *args, **kwargs)
-        if not self.attrs.get("required_score", None):
-            self.attrs["required_score"] = getattr(
+        if not self.attrs.get("data-required-score", None):
+            self.attrs["data-required-score"] = getattr(
                 settings, "RECAPTCHA_REQUIRED_SCORE", None
             )
 
```

Comment:
This could be made backwards compatible by continuing to look for the `required_score` attribute.


(don't forget to `import warnings` at the top of the file, it is not part of this suggestion)
```suggestion
        if not self.attrs.get("data-required-score", None):
            self.attrs["data-required-score"] = getattr(
                settings, "RECAPTCHA_REQUIRED_SCORE", None
            )
        # Support deprecated required_score attribute for backwards compatibility.
        if self.attrs.get("required_score", None):
            warnings.warn("The required_score attribute is deprecated. Use data-required-score instead.", DeprecationWarning, stacklevel=2)
            self.attrs["data-required-score"] = self.attrs["required_score"]
            del self.attrs["required_score"]

```